ingenialink/utils/_utils.py

```diff
@@ -236,10 +236,6 @@ def convert_dtype_to_bytes(data: Union[int, float, str, bytes], dtype: REG_DTYPE
         and not isinstance(data, bytes)
     ):
         raise ValueError(f"Invalid value. Expected values: {VALID_BIT_REGISTER_VALUES}, got {data}")
-    if dtype == REG_DTYPE.BYTE_ARRAY_512:
```

Comment:
Is this change needed?

Reply:
Not really but it does not make sense to have it.

Reply:
because it is already bytes? but maybe in some case, you don't know the register type but you want to convert data to bytes, in this case now it will raise an exception